kotlinpoet/src/main/java/com/squareup/kotlinpoet/ParameterSpec.kt

```diff
@@ -26,6 +26,9 @@ class ParameterSpec private constructor(
   builder: ParameterSpec.Builder,
   private val tagMap: TagMap = builder.buildTagMap()
 ) : Taggable by tagMap {
+  
+  constructor(name: String, type: TypeName, vararg modifiers: KModifier): this(builder(name, type, *modifiers))
```

Comment:
Do we want overloads that accept `java.lang.reflect.Type` and `KClass`, similar to `builder()` overloads?